docs-content/index.html.md.erb

```diff
@@ -149,49 +230,79 @@ Create a policy using the following example as a guide. Save the policy as a YAM
+in the tile configuration. The `pcf-service-broker` host is a member of the `pcf-admin-group` group, which
+owns the `pcf/production` policy, so it has `create` and `update` privileges on that policy as required.
 
-The PCF policy has a `pcf-apps` layer and corresponding host factory. If using Conjur v4, application hosts will be added to the `pcf-apps` layer via the `pcf-apps` host factory when the application is bound to a Conjur service instance.
-Additional policies for individual applications can be appended to this file or loaded in separate YAML files as needed.
+The PCF policy has a `pcf/production-apps` layer and corresponding host factory. If using Conjur v4, application
+hosts will be added to the `pcf/production-apps` layer via the `pcf-apps` host factory when the application is
+bound to a Conjur service instance.
 
 #### Step 2
 
 Load the policy into Conjur as follows (in this example, we are loading it into the root policy):
 
-For Conjur Enterprise (v4):
+**For Conjur Enterprise v4:**
 
 <pre class='term'>$ conjur policy load --as-group security_admin your-pcf-policy-file.yml</pre>
 
-For Conjur Open Source (v5):
+> **NOTE:** You may have to first install the `policy` plugin for the Conjur CLI. If you receive
+> this message when loading the policy:
+>
+> ```
+> $ conjur policy load ...
+> error: Unknown command 'policy'
+> ```
+>
+> Then you may install the `policy` plugin using the command:
+>
+> ```
+> $ conjur plugin install policy
+> Successfully installed conjur-asset-policy-0.13.0
+> 1 gem installed
+> ```
+
+
+**For Conjur Open Source and Conjur Enterprise v5:**
 
 <pre class='term'>$ conjur policy load root your-pcf-policy-file.yml</pre>
 
+Loading this policy the first time will output the API key for the `pcf-service-broker` host. Save
+this key to use when configuring the PCF tile below.
+
 #### Step 3
 
 In the CyberArk Conjur Service Broker for PCF tile configuration:
 
-* For **PCF Conjur Policy Branch ID**, use the fully-qualified ID of the dedicated Conjur policy for PCF (`pcf` in the above example).
-* For **Conjur Login**, use a Conjur host who is a member of the Conjur group that owns the PCF policy (`host/pcf-service-broker` in the example above). The host also needs to be permitted to read its own annotations.
-* For **Conjur API Key**, use that host's API key. The admin who created the role can retrieve the value with this command:
+* For **PCF Conjur Policy Branch ID**, use the fully-qualified ID of the dedicated Conjur policy for PCF
+  (`pcf/production` in the above example). For **Conjur Login**, use a Conjur host who is a member of the
+  Conjur group that owns the PCF policy (`host/pcf-service-broker` in the example above). The host also
+  needs to be permitted to read its own annotations.
 
-<pre class='term'>$ conjur host rotate_api_key -h pcf-service-broker</pre>
+* For **Conjur API Key**, use that host's API key. The admin who created the role can retrieve the value with this command:
```

Comment:
Is it intentional that this remains a separate bullet? It looks like the bullet for `For Conjur Login` got merged into the bullet above